CIP-XXXX/README.md

```diff
@@ -0,0 +1,176 @@
+---
+CIP: ?
+Title: Royalty Datum Metadata
+Authors: Sam Delaney <sam@ikigaitech.org>
+Comments-URI:
+Status: Proposed
+Type: Informational
```

Comment:
```suggestion
Category: Tokens
```
There is no longer a `Type` field; with a new `Category` field set as follows: https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#categories